lightning/webelement.py

```diff
@@ -0,0 +1,203 @@
+    def tag_name(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.TAG_NAME)
+
+    @staticmethod
+    def link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.LINK_TEXT)
+
+    @staticmethod
+    def partial_link_text(selector: str) -> By:
+        return By(expression=selector, strategy=LocatorStrategy.PARTIAL_LINK_TEXT)
+
+
+class Accessibility(dict):
+    def __init__(self, elements_instance: ElementsApi, session_id: str, element_id: str):
+        self._elements_instance = elements_instance
+        self._session_id = session_id
+        self._element_id = element_id
+
+    @property
+    def role(self) -> str:
+        _role = self._elements_instance\
+            .get_element_computed_role(session_id=self._session_id, element_id=self._element_id).value
+        return _role
+
+    @property
+    def label(self) -> str:
+        _label = self._elements_instance\
+            .get_element_computed_label(session_id=self._session_id, element_id=self._element_id).value
+        return _label
+
+    def __repr__(self):
+        _c = dict(role=self.role, label=self.label)
+        return repr(_c)
+
+
+class WebElement(dict):
+
+    def __init__(self, elements_instance: ElementsApi, element_id: str, session_id: str):
+        self._elements_instance = elements_instance
+        self._element_id = element_id
+        self._session_id = session_id
+
+    def accessibility(self) -> Accessibility:
+        _accessibility = Accessibility(elements_instance=self._elements_instance,
+                                       session_id=self.session_id,
+                                       element_id=self.element_id)
+        return _accessibility
+
+    @property
+    def element_id(self) -> str:
+        return self._element_id
+
+    @property
+    def session_id(self) -> str:
+        return self._session_id
+
+    def click(self) -> WebElement:
+        _web_element = self._elements_instance.element_click(session_id=self.session_id,
+                                                             element_id=self.element_id,
+                                                             body=EmptyRequest())
+        return self
+
+    def clear(self) -> WebElement:
+        _web_element = self._elements_instance.element_clear(session_id=self.session_id,
+                                                             element_id=self.element_id,
+                                                             body=EmptyRequest())
+        return self
+
+    def find_all(self, locator: Locator) -> list[WebElement]:
+        _body = FindElementRequest(using=locator.strategy, value=locator.expression)
+        _response: list[FindElementResponseValue] = self._elements_instance.find_elements(session_id=self.session_id,
+                                                                                          body=_body).value
+        # ? raise smth if len(_response) == 0
```

Comment:
Not needed. If response is empty list we return an empty list of elements.

Reply:
updated in 990528b5d10e02f0199fddd9a570dc33e95fa248